packages/@sanity/form-builder/src/inputs/PortableText/Toolbar/Toolbar.tsx

```diff
@@ -32,18 +38,95 @@ function PTEToolbar(props: Props) {
   const editor = usePortableTextEditor()
   const selection = usePortableTextEditorSelection()
   const disabled = !selection
+
+  const toast = useToast()
+
+  const resolveInitialValue = useCallback(
+    (type: Type) => {
+      let isSlow = false
+      const slowTimer = setTimeout(() => {
+        isSlow = true
+        toast.push({
+          closable: true,
+          id: 'resolving-initial-value',
+          status: 'info',
+          title: 'Resolving initial value…',
+        })
+      }, SLOW_INITIAL_VALUE_LIMIT)
+      return resolveInitialValueForType((type as any) as SchemaType)
+        .then((value) => {
+          if (isSlow) {
+            toast.push({
+              closable: true,
+              id: 'resolving-initial-value',
+              status: 'info',
+              duration: 1000,
+              title: 'Initial value resolved',
+            })
```

Comment:
Not sure we need to show this, as the value appears?

Reply:
I found no way to close an existing toast, so this will replace the message in the "Resolving initial value…"-toast and then make sure it's closed.

Reply:
Decreased the duration to 500, so it disappears a bit faster.